.github/workflows/windows.yml

```diff
@@ -56,7 +56,6 @@ jobs:
         uses: lukka/run-vcpkg@v11
         with:
           vcpkgDirectory: "C:/vcpkg" # use VCPKG_INSTALLATION_ROOT of the image
-          doNotUpdateVcpkg: true
           runVcpkgInstall: true
```

Comment:
```suggestion
          runVcpkgInstall: true
          vcpkgGitCommitId: "c82f74667287d3dc386bce81e44964370c91a289" # 2024.09.30
```

Reply:
It seems this action can auto find vcpkg git commit id from vcpkg-configuration.json file.